src/main/components/HeaderBar.jsx

```diff
@@ -7,19 +10,19 @@ const HeaderBar = (props) => {
 
   return (
     <div>
-      <p>Headers</p>
-      <form>
-      <select name='Authentication' id='headerTypeInput' multiple={false} value={header}
+      {/* <p>Headers</p> */}
+      <Form bordered={false}>
```

Comment:
I don't particularly care, but why did we capitalize the first letter?  Never seen this before

Reply:
It’s to indicate that it’s a component (in this case a styled component) rather than the HTML element (just convention from React/styled-components docs: https://www.styled-components.com/docs/basics#getting-started)